Okta.AspNet/TokenExchanger.cs

```diff
@@ -32,8 +31,15 @@ public TokenExchanger(OktaMvcOptions options, string issuer, ConfigurationManage
         public async Task ExchangeCodeForTokenAsync(AuthorizationCodeReceivedNotification response)
         {
             var openIdConfiguration = await _configurationManager.GetConfigurationAsync().ConfigureAwait(false);
-            var tokenClient = new TokenClient(openIdConfiguration.TokenEndpoint, _options.ClientId, _options.ClientSecret);
-            var tokenResponse = await tokenClient.RequestAuthorizationCodeAsync(response.Code, _options.RedirectUri).ConfigureAwait(false);
+            var httpClient = new HttpClient();
```

Comment:
The usual advice I see for .NET is to create one, singleton instance of `HttpClient` and re-use that for all threads. Any reason we are not doing that here?

Reply:
I missed that here, thanks for bringing this up 👍